pkg/node/node.go

```diff
@@ -438,6 +451,58 @@ func (n *Node) publishSignature(
 	}
 }
 
+func (n *Node) waitSignerPublicationDelay(
```

Comment:
`waitSignaturePublicationDelay`? We are publishing *signature* so it's *signature* publication delay IMO.